app/models/patient.rb

```diff
@@ -1,2 +1,4 @@
 class Patient < ApplicationRecord
+  has_many :cases
+  belongs_to :users, through: :cases
```

Comment:
Line 3 should be has_many :users, through: :cases in my opinion

Reply:
i was thinking using has_many. then I thought, since we will not has patient as user. 
Because only doctor can create patient file. Maybe one patient to many doctor relationship will not be needed here? what do you think?

Reply:
Even though we do not have a patient as a user, i think we need to specify it is a many to many relationship though the cases table.